Source/WebCore/platform/Scrollbar.h

```diff
@@ -48,6 +48,7 @@ class Scrollbar : public Widget {
     void offsetDidChange();
 
     static int pixelsPerLineStep() { return 40; }
```

Comment:
Can you make this one private to ensure callers have to pass the int parameter, or is it still used elsewhere? (Removing the option to not pass the parameter would discourage misuse.) Actually, it's probably cleaner if we have just one function here. Also, I think we should still change this to default to 120 (at least for Windows and WPE) and use 40 only for Cocoa. So I would do something like:

```
int Scrollbar::pixelsPerLineStep(int viewWidthOrHeight)
{
#if PLATFORM(GTK)
    if (const char* scrollStep = getenv("WEBKIT_SCROLL_STEP")) {
        int value;
        if (sscanf(scrollStep, "%d", &value) == 1 && value > 0)
            return value;
    }

    if (viewWidthOrHeight > 0)
        return std::pow(viewWidthOrHeight, 2. / 3.);
#elif PLATFORM(COCOA)
    UNUSED_PARAM(viewWidthOrHeight);
    return 40;
#else
    UNUSED_PARAM(viewWidthOrHeight);
    return 120;
#endif
}
```

Reply:
This is used in other places where we don't want to pass the view size. I don't think we should change the behavior of other ports in this patch, for wpe we should probably do the same than gtk, but in a follow up after discussing with other wpe maintainers.